tracer/test/snapshots/PipesXUnitTests.SubmitTraces_packageVersion=all.verified.txt

```diff
@@ -0,0 +1,1015 @@
+[
```

Comment:
Presumably these should be identical to existing, regardless of transport? If so then I strongly suggest reusing the same files for all the tests to avoid discrepencies - you'll never notice if named pipes is getting something wrong here I'd wager 😉

Reply:
True, done in https://github.com/DataDog/dd-trace-dotnet/pull/5634/commits/5f70ddf7aac47e740c4b9a0fe61ae5728af8a310